cmd/signozcollector/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Builder stage
-FROM golang:1.19-alpine as build
+FROM golang:1.19 as build
```

Comment:
@srikanthccv can you revert this particular change?

`push-workflow` is failing because the rest of the steps in Dockerfile assumes the base image to be alpine.

Reply:
Anyways, this should be resolved when we merge PR #136